Description: Fix tox.ini
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2024-05-02

--- python-sure-2.0.1+git.2023.02.06.3aef950b7c.orig/tox.ini
+++ python-sure-2.0.1+git.2023.02.06.3aef950b7c/tox.ini
@@ -5,12 +5,3 @@
 
 [tox]
 envlist = py27, pypy, py34, py35, py36, py37
-
-[testenv]
-commands = nosetests --rednose -vv --with-coverage --cover-package=sure
-deps =
-    six
-    mock
-    nose
-    rednose
-    coverage
